src/store/selectors/groups.js

```diff
@@ -28,7 +28,7 @@ export const selectGroups = (state) => {
 
 
 export const selectGroupById = (state, { groupId }) => {
-  return state.groups[groupId]
+  return state.groups?.[groupId]
```

Comment:
the groups object itself should always be defined. if its not then we need to fix initialState.

Reply:
I may have been overzealous here.

Reply:
we should only null check what can be null :P